__tests__/component/form/CreateCustomerForm.test.tsx

```diff
@@ -108,20 +115,17 @@ test("onCancel is invoked", async () => {
 })
 
 test("a required field cannot be missing", async () => {
-    const submitTimeout = 100
-    for (const field of Object.keys(testCustomerRequiredFields)) {
-        const form = render(<CreateCustomerForm />)
-        const customerMissingRequired = _.omit(testCustomerAllFields, field)
-
-        /* eslint-disable no-await-in-loop */
-        await fillAndSubmitForm(customerMissingRequired as Customer)
-        await new Promise((resolve) =>
-            setTimeout(() => resolve(null), submitTimeout)
-        )
-        /* eslint-enable */
-
-        expect(pathSpy.callCount).toEqual(0)
-        expect(bodySpy.callCount).toEqual(0)
-        form.unmount()
-    }
+    await Promise.all(
```

Comment:
Ymmärtääkseni Promiseja käsitellään joko async/await tai .then() -tyyliin. Molempia ei tarvita 🙂

Reply:
Tämä pitää paikkansa! Oli joku katkos järjen juoksussa tuossa kohtaa.. Poistan noi turhat await:it